src/main/java/io/debezium/connector/vitess/connection/VitessConnectionUtils.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.vitess.connection;
+
+import io.debezium.annotation.Immutable;
+
+@Immutable
+public class VitessConnectionUtils {
+    /** The types of vitess tablet. */
+    public enum TabletType {
```

Comment:
Looks like this could be a top-level type, rather than being nested into the otherwise empty `VitessConnectionUtils`?

Reply:
Yes, looks like this could be a top-level Enum indeed